### DISCUSSION TITLE
Termux does it look like Disk Operating System

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [termux/termux-packages#11245] [Q&A] [answered]
Termux comparable  MS-DOS FREEDOS

**Comment by ExtinctDrift19** [comment]
what is the difference between termux and DOS

**Reply by TermuxMonet** [reply]
Unlike DOS, Termux supports multitasking and is a multi user operating system (like all other linux).
DOS cannot run multitasks.

**Reply by ExtinctDrift19** [reply]
installation app the inside of a termux is similar to DOS

**Reply by ExtinctDrift19** [reply]
similar to installing an application in the DOS EXE FILE

**Reply by kcubeterm** [reply]
>  and is a multi user operating system (like all other linux). 

Termux don't support multi-user, limitations comes from android side.

**Comment by 2096779623** [comment]
> Termux is an Android terminal emulator and Linux environment app that works directly with no rooting or setup required. A minimal base system is installed automatically - additional packages are available using the APT package manager.

https://wiki.termux.com/wiki/Getting_started

https://wiki.termux.com/wiki/FAQ

**Comment by sylirre** [comment, marked as answer]
The main difference is... Termux is not operating system!

Termux is just an application that intercepts standard input/output streams created by programs and interprets the received data, basically what any terminal emulator does. It also uses a custom Linux userland instead of what Android provides, so you can have Bash, Python, Nginx and other stuff which is not part of the Android operating system.

**Reply by ExtinctDrift19** [reply]
Termux let's compare with DOS
